<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fournisseurs</class>
 <widget class="QDialog" name="Fournisseurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>881</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>68</red>
         <green>68</green>
         <blue>68</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>home</string>
    </attribute>
    <widget class="QLabel" name="bg_home">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>671</width>
       <height>401</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../Downloads/black-gold-marble-texture_125540-835.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="history">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 0);</string>
     </property>
     <property name="text">
      <string>Historique de l'application</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trier">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 0);</string>
     </property>
     <property name="text">
      <string>Trier Fournisseur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recherche">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 0);</string>
     </property>
     <property name="text">
      <string>Rechercher Fournisseur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mail">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 0);</string>
     </property>
     <property name="text">
      <string>Mailling </string>
     </property>
    </widget>
    <widget class="QPushButton" name="maps">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>190</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 0);</string>
     </property>
     <property name="text">
      <string>Géolocalisation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>791</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>bg_home</zorder>
    <zorder>history</zorder>
    <zorder>trier</zorder>
    <zorder>recherche</zorder>
    <zorder>mail</zorder>
    <zorder>maps</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ajouter fournisseur</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>741</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>ajout</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(206, 197, 64);
font: 75 11pt &quot;MS Shell Dlg 2&quot;; </string>
      </property>
      <property name="text">
       <string>nom du fournisseur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;

color: rgb(206, 197, 64);</string>
      </property>
      <property name="text">
       <string>identifiand du fournisseur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(206, 197, 64);
font: 75 11pt &quot;MS Shell Dlg 2&quot;; </string>
      </property>
      <property name="text">
       <string>numéro du téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numtel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>220</red>
            <green>220</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 0);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(206, 197, 64);
font: 75 11pt &quot;MS Shell Dlg 2&quot;; </string>
      </property>
      <property name="text">
       <string>categorie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Le_cat">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(206, 197, 64);
font: 75 11pt &quot;MS Shell Dlg 2&quot;; </string>
      </property>
      <property name="text">
       <string>Qualité du produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Le_qal">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 0);</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(206, 197, 64);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(206, 197, 64);</string>
      </property>
      <property name="text">
       <string>choisir un identifiant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 0);</string>
      </property>
      <property name="text">
       <string>suprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_img_ajt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>691</width>
        <height>411</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>lb_img_ajt</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_id</zorder>
     <zorder>le_numtel</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>label_6</zorder>
     <zorder>Le_cat</zorder>
     <zorder>label_8</zorder>
     <zorder>Le_qal</zorder>
     <zorder>pb_modifier</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_10</zorder>
     <zorder>pb_supp</zorder>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>791</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>afficher fournisseur</string>
    </attribute>
    <widget class="QLabel" name="bg_afficher">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../Downloads/black-gold-marble-texture_125540-835.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>791</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tabfournisseur">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>501</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trie_ref">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trie par nom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trie_cat">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trie par categorie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trie_qual">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trie par qualité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recher_edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>561</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>bg_afficher</zorder>
    <zorder>tabfournisseur</zorder>
    <zorder>trie_ref</zorder>
    <zorder>trie_cat</zorder>
    <zorder>trie_qual</zorder>
    <zorder>recher_edit</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTableView" name="tablerecherc">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>641</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>206</red>
         <green>197</green>
         <blue>64</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: rgb(206, 197, 64);
font: 22pt &quot;Palatino Linotype&quot;</string>
   </property>
   <property name="text">
    <string>Fournisseur </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
